package.json

```diff
@@ -32,13 +41,29 @@
     "url": "https://github.com/bigbinary/neeto-hotkeys/issues"
   },
   "homepage": "https://github.com/bigbinary/neeto-hotkeys#readme",
+  "dependencies": {
+    "mousetrap": "^1.6.5",
+    "mousetrap-global-bind": "^1.1.0",
+    "platform": "^1.3.6",
+    "ramda": "^0.29.1",
+    "react": "^18.2.0"
+  },
```

Comment:
If you are serving a bundled package, why do you have entries in dependencies?

Reply:
Please watch this: https://vimeo.com/manage/videos/885502289/2d2be16f29